app/views/search.scala.html

```diff
@@ -10,10 +10,10 @@
 }
 
 @if( results.isEmpty ){
-    <p>No results found.</p>
+    <p><Strong>No results found.</Strong> Try different keywords and we'll see if they appear in any of our questions, hints, or solutions.</p>
```

Comment:
The query also searches for the keyword in `topics` field of questions collection.